apis/metal3.io/v1alpha1/baremetalhost_validation.go

```diff
@@ -309,3 +314,34 @@ func validateRebootAnnotation(rebootAnnotation string) error {
 
 	return nil
 }
+
+func validateCrossNamespaceSecretReferences(hostNamespace, hostName, fieldName string, ref *corev1.SecretReference) error {
```

Comment:
Document the function

Reply:
Hmm. OK I will, but imo these functions are quite self-explanatory, and the other validate...() func in this files don't seem to have docstring either.

Reply:
We are missing the linter for that, but want to enable it like in CAPM3, so no need to add future work. It is good to document them right away, it is the proper thing to do.